python/modules/IcePy/Operation.cpp

```diff
@@ -2179,7 +2152,7 @@ IcePy::SyncBlobjectInvocation::invoke(PyObject* args, PyObject* /* kwds */)
     }
 
     PyObjectHandle modeValue = getAttr(mode, "value", true);
-    Ice::OperationMode sendMode = (Ice::OperationMode) static_cast<int>(PyLong_AsLong(modeValue.get()));
+    Ice::OperationMode op_mode = (Ice::OperationMode) static_cast<int>(PyLong_AsLong(modeValue.get()));
```

Comment:
It should be `opMode` or `mode` as per our C++ naming conventions

Reply:
Fixed!